CityOfDetroit/COD-Design-System
# Update UXDS to match detroitmi starterkit themes

## Part of #119 

## This PR

A handful of changes to make the stylesheets of the UXDS consistent with those in the detroitmi starterkit so that we may begin reusing the stylesheets in the UXDS in the detroitmi starter kit theme instead of duplicating.

Specific changes:
* Upgrade sass to 1.71+
* Stop using relative filepaths to import bootstrap and rely on include paths with node modules in webpack and sass command line instead.
* Implement additional color utilities for accents based on city yellow and pale green

## Testing

* Check that storybook is not impacted in chromatic.
* Add additional storybook params to test additional accent colors.

![image](https://github.com/CityOfDetroit/COD-Design-System/assets/143553259/5c18b322-1abd-447f-9afc-c4eba8d6d61a)
